python/py-readme_renderer/Portfile

```diff
@@ -2,32 +2,43 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-PortGroup           github 1.0
-
-github.setup        pypa readme_renderer 26.0
 
 name                py-readme_renderer
-platforms           darwin
+version             30.0
+revision            0
 supported_archs     noarch
 license             apache
 maintainers         {aronnax @lpsinger} openmaintainer
 
 description         Safely render long_description/README files in Warehouse
-long_description    ${description}
+long_description    {*}${description}
+
+homepage            https://github.com/pypa/readme_renderer
 
-checksums           rmd160  67451b077be105f5f1beb1ca997249d54dd6f86f \
-                    sha256  9852c2c926d82444a99b818042e9d296616db3bb9084a14aef1940161dd502d8 \
-                    size    23800
+checksums           rmd160  017c4292b1f7b7cc51b9e5ddc89769cd547f55bb \
+                    sha256  8299700d7a910c304072a7601eafada6712a5b011a20139417e1b1e9f04645d8 \
+                    size    27065
 
-python.versions     27 36 37 38
+python.versions     27 37 38 39
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
+    if {${python.version} eq {27}} {
+        version     28.0
+        revision    0
+
+        checksums   rmd160  2e75426d44eda0b2861d4906a9aa50e0a642e1e5 \
+                    sha256  6b7e5aa59210a40de72eb79931491eaf46fefca2952b9181268bd7c7c65c260a \
+                    size    27370
+    }
+    
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
     depends_run-append \
-        port:py${python.version}-bleach \
-        port:py${python.version}-docutils \
-        port:py${python.version}-pygments \
-        port:py${python.version}-six
+                    port:py${python.version}-bleach \
+                    port:py${python.version}-docutils \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-six
```

Comment:
I suspect the latest version doesn't use `six` anymore as it's PY3 only - if that's the case please move this dependency within the `if {${python.version} eq {27}} {` block

Reply:
Thanks, done.